modules/ting/ting.entities.inc

```diff
@@ -60,6 +60,13 @@ class TingEntity extends DingEntity {
   public $classification = DingEntityBase::NULL;
   public $isbn = DingEntityBase::NULL;
 
+  /**
+   * The embedded search-provider specific object.
+   *
+   * @var \Drupal\ting\TingObjectInterface
+   */
+  public $reply;
```

Comment:
Yup, had a back and forth with @kasperg about it - I still hope we'll find a better name before I'm done, but  lets see :)